pkg/server/config.go

```diff
@@ -491,6 +493,52 @@ func (c Config) installApiServices() error {
 	return nil
 }
 
+func (c Config) CleanupApiServices() error {
+	apiRegClient, err := apiregistrationv1client.NewForConfig(c.Rest)
+	if err != nil {
+		return err
+	}
+	var finalErr error
+
+	if c.APIServices.StoreReports {
+		if err := c.toLocalApiService(c.APIServices.wgpolicyApiService.GetName(), *apiRegClient); err != nil {
+			finalErr = multierror.Append(finalErr, err)
+		}
+	}
+	if c.APIServices.StoreEphemeralReports {
+		if err := c.toLocalApiService(c.APIServices.v1ReportsApiService.GetName(), *apiRegClient); err != nil {
+			finalErr = multierror.Append(finalErr, err)
+		}
+	}
+	if c.APIServices.StoreOpenreports {
+		if err := c.toLocalApiService(c.APIServices.openreportsApiService.GetName(), *apiRegClient); err != nil {
+			finalErr = multierror.Append(finalErr, err)
+		}
+	}
+
+	return finalErr
+}
+
+func (c Config) toLocalApiService(apiSvcName string, client apiregistrationv1client.ApiregistrationV1Client) error {
+	_, err := client.APIServices().Get(context.TODO(), apiSvcName, metav1.GetOptions{})
+	if err != nil && !errors.IsNotFound(err) {
+		return err
+	}
+	patch := []byte(`{
+		"spec": {
+		  "service": null,
+		  "insecureSkipTLSVerify": false
+		}
+	  }`)
+
+	_, err = client.APIServices().Patch(context.TODO(), apiSvcName, types.MergePatchType, patch, metav1.PatchOptions{})
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
(Nitpicking from my side, feel free to disregard) I guess we might as well not do an `if` at all and simply `return err` ? 😅

```suggestion
	return err
```